pkg/image/decompress.go

```diff
@@ -40,15 +41,23 @@ func UnpackData(filename string, src io.ReadCloser) (io.ReadCloser, error) {
 // return a ReadCloser.  It is up to the caller of DecompressData to close the returned stream.
 // If no compression is detected, it is considered a 'noop' and the original stream is returned.
 // Returns trimmed filename string and gzip Reader if gzip compression was used.
-func DecompressData(filename string, src io.ReadCloser) (string, io.ReadCloser, error) {
+func DecompressData(filename string, src io.ReadCloser) (fn string, rc io.ReadCloser, err error) {
```

Comment:
Why use named returns here and not elsewhere?  I'm okay with going either way, so long as it's consistent.

Reply:
It was to save a declaration of `var err error` within the scope of the outer switch. Go doesn't permit one returned value to be named while the other returned values are unnamed, so I named them all. 
I'll declare `err` explicitly.